puara_gestures.cpp

```diff
@@ -73,7 +73,7 @@ void PuaraGestures::updateJabShake() {
 }
 
 void PuaraGestures::updateOrientation() {
-  orientation.update();
+  orientation.update(0.01);
```

Comment:
Nice. I'd add a comment on the expected unit (seconds, milliseconds?)

Reply:
This is just the weight vs the previous orientation, no unit.

Reply:
Then definitely needs a comment. At first glance and without following the method I was expecting an update rate (freq) value

Reply:
added!